app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -3211,6 +3211,10 @@
   },
   {
     "descriptors": [
+      {
+        "defaultMessage": "Hello #Mastodon!",
```

Comment:
What about the common `#Introduction` tag with a little guidance on those posts?